prql-compiler/tests/integration/connection.rs

```diff
@@ -242,45 +243,71 @@ impl DBConnection for MysqlConnection {
                     Value::UInt(v) => v.to_string(),
                     Value::Float(v) => v.to_string(),
                     Value::Double(v) => v.to_string(),
-                    typ => unimplemented!("mysql type {:?}", typ),
+                    typ => bail!("mysql type {:?}", typ),
                 };
                 columns.push(value);
             }
             vec.push(columns);
         }
-        vec
+        Ok(vec)
     }
 
     fn import_csv(&mut self, csv_name: &str, runtime: &Runtime) {
-        self.run_query(&format!("LOAD DATA INFILE '/tmp/chinook/{csv_name}.csv' INTO TABLE {csv_name} FIELDS TERMINATED BY ',' OPTIONALLY ENCLOSED BY '\"' LINES TERMINATED BY '\n' IGNORE 1 ROWS;"), runtime);
+        // hacky hack for MySQL
```

Comment:
```suggestion
        // hacky hack for MySQL
        // TODO: could use https://docs.rs/tempfile/latest/tempfile/struct.NamedTempFile.html
```

(I don't think it's worth doing now, this is more than fine atm)

Reply:
tempfile won't work because it uses the tmp directory of the OS. But the new file must exist in that directory because it is "mounted" to the docker container

Reply:
Sorry, you're right. I had thought that possibly it was sending the _contents_ of the file, but it's indeed loading from the file itself.